### ISSUE TITLE
Broken Python doc generation

### ISSUE THREAD

**Issue by jfcherng** [20Tauri/DoxyDoxygen#136]
I have the Python docstring prefered.

```js
    // List of possible styles for comments (first match is better).
    "preferred_comments_styles": [
        [
            "/**",
            " *"
        ],
        [
            ["/*", "-", "*//**"],   // <=> /*-------------------*//**
            " *"
        ],
        [
            ["//", "-"],            // <=> //------------------------
            "///"
        ],
        // [
        //     [ "//", "/", "" ],
        //     [ "//", "/", " ", " ", "/", "//" ],
        //     [ "", "/", "//" ]
        // ],
        [
            "\"\"\"",            // Python docString
            "",
            "\"\"\""
        ],
        [
            "##",
            "##"
        ],
        [
            ["#", "-"],             // <=> #-------------------------
            "##"
        ]
    ],
```

## Before

```py
def get_timestamp() -> float:
    return time.time()
```

## After

The return type hinting is split.

```py
def get_timestamp() ->
 """
@brief Gets the timestamp.

@return The timestamp.
""" float:
    return time.time()
```

## Expected

```py
def get_timestamp() -> float:
    """
    @brief Gets the timestamp.

    @return The timestamp.
    """
    return time.time()
```

or

```py
def get_timestamp() -> float:
"""
@brief Gets the timestamp.

@return The timestamp.
"""
    return time.time()
```

**Comment by 20Tauri** [comment]
Thanks for reporting,

This has been fixed in 0.72.1 (just released).

**Comment by jfcherng** [comment]
thanks. confirm fixed.
